kotlin-extensions/src/main/kotlin/kotlinx/js/timers/interval.kt

```diff
@@ -0,0 +1,10 @@
+package kotlinx.js.timers
+
+external fun setInterval(
+    callback: () -> Unit,
+    ms: Double = definedExternally,
```

Comment:
Are you sure that we need `Double` here?
https://stackoverflow.com/questions/7286178/does-settimeout-in-javascript-accept-real-floating-point-delay-times